el-gr/pyqt-gr.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+category: tool
+tool: PyQT
+filename: learnpyqt.py
+contributors:
```

Comment:
Please add your name to `translators` section. Also, please add this line: `lang: el-gr`